src/test/java/com/thomashan/coup/PlayerCardsTest.java

```diff
@@ -0,0 +1,105 @@
+package com.thomashan.coup;
+
+import com.thomashan.collection.immutable.ImmutableList;
+import org.junit.jupiter.api.Test;
+
+import static com.thomashan.coup.Card.AMBASSADOR;
+import static com.thomashan.coup.Card.ASSASSIN;
+import static com.thomashan.coup.Card.CAPTAIN;
+import static com.thomashan.coup.Card.DUKE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.junit.jupiter.api.Assertions.assertNotSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class PlayerCardsTest {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This class has too many methods, consider refactoring it.](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)